app/src/main/java/deakin/gopher/guardian/model/MedicalDiagnostic.kt

```diff
@@ -0,0 +1,77 @@
+package deakin.gopher.guardian.model
+
+class MedicalDiagnostic {
```

Comment:
You should use a Kotlin data class. This should allow you to remove all the setters and getters since those are all implicitly generated. 

Take a look at the GP model for reference.